src/App.js

```diff
@@ -190,7 +190,7 @@ let App = () => {
 
 
   // fluid="xxl" is only supported in Bootstrap 5
-  return <Container fluid>
+  return <Container>
```

Comment:
Seems like the reasoning behind not doing this originally (going by @pl4nty's comment at least) was not being able to selectively apply it to large displays, as calendars are a very width-heavy interface so having the extra width was broadly worth it. Possibly instead of this, it could be worth instead implementing a custom breakpoint property and doing the fluid="xxl" yourself? Something like this:
```js
// -- DIY fluid="xxl" implementation
  // Window dimensions
  const getWindowDimensions = () => ({width: window.innerWidth, height: window.innerHeight});
  const [windowDimensions, setWindowDimensions] = useState(getWindowDimensions());

  useEffect(() => {
    function handleResize() {
      setWindowDimensions(getWindowDimensions());
    }

    window.addEventListener('resize', handleResize);
    return () => window.removeEventListener('resize', handleResize);
  }, []);

  // DIY Breakpoint
  const lessThanXxl = windowDimensions.width < 1200;
```
Which would let you instead of deleting fluid entirely, say:
```jsx
  return <Container fluid={lessThanXxl}>
  ```

Reply:
Spot on @ashleylamont, the xl breakpoint was too small and I didn't want to worry about a custom breakpoint at the time. But it might be the ideal solution to this problem.

Reply:
Unless we move to v5 but fullcalendar doesn't support that right?

Reply:
Yeah, someone wrote fullcalendar/fullcalendar#6625 but the maintainers are pretty slow to merge

Reply:
I dislike the idea of writing something custom for this. Feels unnecessary to clutter the codebase with something that is supported by an upcoming version of bootstrap, with a pending merge.

I can see why there would be a preference to use "xxl", but I personally would opt for using "xl" in the interim and upgrading to "xxl" when the aforementioned merge request goes through. 

Could you elaborate on why you feel the "xl" breakpoints is insufficient? I think it should be fine for achieving full-width on mobile and a less claustrophobic feeling on desktop devices.

Reply:
> I can see why there would be a preference to use "xxl", but I personally would opt for using "xl" in the interim and upgrading to "xxl" when the aforementioned merge request goes through.

have reduced this request to just be a change from `fluid` to `xl`

Reply:
> Feels unnecessary to clutter the codebase with something that is supported by an upcoming version of bootstrap, with a pending merge.

I'm personally not too concerned about cluttering the codebase in this way - as long as if we make a component in another file with a good name and add a comment explaining why it needs to exist. The goal is not to minimise the amount of code but to reduce the cognitive burden of understanding it.

I wouldn't count on a pending merge getting merged anytime soon, if at all. If it does get merged then we can clean it up after. We already have a monkey patch for another issue that's been around for a year and a half here: https://github.com/anucssa/anutimetable/blob/97ffc8cad03becea1a2126a7ec17727bdaa494ee/src/Calendar.js#L16-L26

Reply:
@talidemestre that dependency PR probably won't be merged for 6+ months unfortunately. I use an ultrawide so did a fair bit of testing, pretty sure xl looked bad but it was one of my first commits on the v2 rewrite so I'm happy to have this discussion.

Agree with your point on clutter, and the codebase isn't ideal - but we are planning a state refactor, and 15 LoC is little in the scheme of things. Plus custom breakpoints are a pretty common pattern (especially compared to some of my awful imperative state handling rip).

Reply:
Yeah xl is not usable (top is staging, bottom is prod). Admittedly, my ultrawide aspect ratio makes it look worse, but still a problem even on 3:2 let alone 16:9.

![image](https://user-images.githubusercontent.com/21111317/149937619-2e254753-cc1b-44a9-89c5-c6c0a30280a3.png)

![image](https://user-images.githubusercontent.com/21111317/149937762-fa9e960b-fa83-4902-ab72-48cfc9098667.png)

Reply:
Definitely I reckon the second is much better. Even if the wider view is a bit excessive on larger displays, it's still far, far more readable and usable. Possibly a max-width could work if we really want to, but tbh that's probably about the extent of what we'd need, since the calendar is so wide already.

Reply:
I looked into it, but honestly even on a 34" I think it's fine. The ultrawide makes staging look worse imo (those borders are thicc).

I reckon we still need to do something about width on mobile though. Agenda view is a workaround, but the week view is a vertical pancake and the rows are tall to compensate. So users have to scroll and read very overwrapped text.